hc/api/management/commands/sendalerts.py

```diff
@@ -48,7 +48,9 @@ def handle_many(self):
         if repeat_list_approved:
             checks = (
                 list(
-                    running_checks.iterator()) +
+                    going_down.iterator()) +
+                list(
+                    going_up.iterator()) +
                 list(repeat_list_approved))
         else:
```

Comment:
Add the going up and going down in the else statement also.

Reply:
@archbishop12 I have made the additions to the else block as pointed put